.github/workflows/qa.yml

```diff
@@ -40,8 +40,16 @@ jobs:
           docker build "${{ matrix.version }}" \
             --pull \
             --tag "sonar-scanner-cli:${{ matrix.tag }}"
-      - name: Test image
-        run: ./run-tests.sh "sonar-scanner-cli:${{ matrix.tag }}"
+
+      - name: Setup BATS
```

Comment:
Why do we need to run the tests twice? I suggest we remove the tests from this workflow, as we have now them in cirrus (this also applies to the other tasks related to the test and the image scanning).

Reply:
It was a benchmark. But I can remove it now.